website/docs/reference/global-configs/logs.md

```diff
@@ -118,9 +118,22 @@ Supply the `-q` or `--quiet` flag to `dbt run` to show only error logs and suppr
 ```text
 dbt --quiet run
 ...
-
 ```
 
+### dbt list logging
+
+In [dbt version 1.5](/docs/dbt-versions/core-upgrade/upgrading-to-v1.5#behavior-changes), there have been changes to the logging behavior of the [dbt list](/reference/commands/list) command. In version 1.5, the default behavior has been updated, and it now includes `INFO` level logs by default
+
+Previously, the list command (and only the list command) had `WARN`-level stdout logging, to support piping its results to [`jq`](https://jqlang.github.io/jq/manual/), a file, or another process.
```

Comment:
Do we need this ☝🏻 paragraph if we move the bits about piping results to the next paragraph?